### ISSUE TITLE
AppCrash on hooking 32 bit process on Windows 10 64 bit with CSharpConsole

### ISSUE THREAD

**Issue by kozhevnikov** [nektra/Deviare2#16]
I'm playing around with CSharpConsole from precompiled [Deviare 2.8.0 release](https://github.com/nektra/Deviare2/releases) under a [Windows 10](https://www.microsoft.com/en-gb/software-download/windows10ISO) 64 bit VM and while `CSharpConsole64.exe` can hook 64 bit process like `notepad.exe` no problem, `CSharpConsole.exe` crashes target 32 bit process like `notepad++.exe`, `devenv.exe`, etc. Is 32 bit hooking supported under Windows 10?

```
Faulting application name: notepad++.exe, version: 6.9.2.0, time stamp: 0x573b9c9e
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x00d90030
Faulting process id: 0x10fc
Faulting application start time: 0x01d21359b0a69664
Faulting application path: C:\Program Files (x86)\Notepad++\notepad++.exe
Faulting module path: unknown
Report Id: c34e8c31-c38b-402c-834c-3df70f833943
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by scnale** [comment]
We have reproduced this and we are taking a look at it now.

**Comment by kozhevnikov** [comment]
Thanks! On an unrelated note, any chance you would have a second to help with [this question](http://stackoverflow.com/q/39605936/384478) please?

**Comment by mxmauro** [comment]
Hi @kozhevnikov , as a quick answer, please comment/remove the call to `BuildOriginalNtCalls` in `Source\Common\DynamicAPIs.cpp` while we code the final patch.

This should fix the crash.

Regards.

**Comment by scnale** [comment]
Fixed in cac444dce62e5c8e8a.
